rotkehlchen/tests/db/test_db.py

```diff
@@ -1041,7 +1041,7 @@ def test_timed_balances_primary_key_works(user_data_dir):
     assert 'Adding timed_balance failed' in str(exc_info.value)
 
     balances = db.query_timed_balances(asset=A_BTC)
-    assert len(balances) == 1
+    assert len(balances) == 0
```

Comment:
Nice catch. Can you also edit this test's docstring? It's essentially testing that it will fail to add two timed balances with the same primary key (category, time, asset)